server/pulp/server/async/app.py

```diff
@@ -87,6 +84,20 @@ def initialize_worker(sender, instance, **kwargs):
         get_resource_manager_lock(sender)
 
 
+@worker_shutdown.connect
+def shutdown_worker(signal, sender):
+    """
+    Called when a worker is shutdown. So far, this just cleans up the database
```

Comment:
The first sentence of a docstring should be a 1-line, then whitespace, then a multi-line paragraph which is optional. See 257 for convention details. In this case this second sentence can just be moved to its own paragraph. A similar change can be made for the other docstring.